components/script/dom/bindings/htmlconstructor.rs

```diff
@@ -166,6 +166,8 @@ where
             // Step 8.4
             element.set_custom_element_definition(definition.clone());
 
+            element.done_creating();
```

Comment:
This seems to have become substeps of step 9, could you renumber the steps and add a todo for the now missing part?